lib/Core/ExecutorUtil.cpp

```diff
@@ -153,6 +153,8 @@ namespace klee {
     case Instruction::FCmp:
       assert(0 && "floating point ConstantExprs unsupported");
     }
+    
+    assert(0 && "unsupported expression in evalConstantExpr");
```

Comment:
In this instance, I will change it, as we have an `assert(0)` above, so this is clearly unreachable.